### DISCUSSION TITLE
漏洞 CVE-2024-29831、CVE-2024-30188

### DISCUSSION THREAD

**Discussion by invainX** [apache/dolphinscheduler#16438] [Q&A]
请问 漏洞 CVE-2024-29831、CVE-2024-30188，是哪个mr合入解决的呢？

**Comment by lishiyucn** [comment]
我也是来找这个的，升级牵扯太大，需要源码修复

**Reply by stephenking30** [reply]
> 我也是来找这个的，升级牵扯太大，需要源码修复

这个问题你这边找到解决方案了么

**Comment by stephenking30** [comment]
> 请问 漏洞 CVE-2024-29831、CVE-2024-30188，是哪个mr合入解决的呢？

我也在找这个mr，有清楚的麻烦告知一下，直接大版本升级不太现实

**Comment by LifeHackerBee** [comment]
https://github.com/apache/dolphinscheduler/commit/ca93aa8e260b4e227c054d42aca20b272574e851

**Reply by invainX** [reply]
这个mr应该是修复了 CVE-2024-29831

**Comment by invainX** [comment]
https://github.com/apache/dolphinscheduler/commit/88a8f06b1d42da00faf992b3542624f4c3590563 应该是修复了 CVE-2024-30188

**Reply by stephenking30** [reply]
> [88a8f06](https://github.com/apache/dolphinscheduler/commit/88a8f06b1d42da00faf992b3542624f4c3590563) 应该是修复了 [CVE-2024-30188](https://github.com/advisories/GHSA-4vv4-crw4-8pcw)

你好，你这边能复现出来这个漏洞问题么？

**Reply by invainX** [reply]
还没复现出
